src/main/java/ru/l3r8y/ValidateMojo.java

```diff
@@ -54,19 +55,35 @@ public final class ValidateMojo extends AbstractMojo {
 
     /**
      * The fail on error.
+     *
      * @checkstyle MemberNameCheck (6 lines).
      */
     @SuppressWarnings("PMD.ImmutableField")
     @Parameter(defaultValue = "true")
     private boolean failOnError = true;
 
+    /**
+     * Length of Class Name.
+     *
+     * @see ru.l3r8y.rule.LongClassNameCheck
+     * @checkstyle MemberNameCheck (6 lines).
+     */
+    @SuppressWarnings("PMD.ImmutableField")
+    @Parameter
+    private int okClassNameLength = 13;
```

Comment:
@h1alexbel i think this parameter can be configurable for each project, can you do it?

```xml
<maxClassNameLen>13<maxClassNameLen>
```